src/Breadcrumb/index.jsx

```diff
@@ -48,12 +49,18 @@ Breadcrumbs.propTypes = {
   /** allows to add a custom function to be called `onClick` of a breadcrumb link.
    * The use case for this is for adding custom analytics to the component. */
   clickHandler: PropTypes.func,
+  /** The `Breadcrumbs` style variant to use. */
+  variant: PropTypes.oneOf(['light', 'dark']),
+  /** The `Breadcrumbs` mobile variant view. */
+  isMobile: PropTypes.bool,
 };
 
 Breadcrumbs.defaultProps = {
   activeLabel: undefined,
   spacer: undefined,
   clickHandler: undefined,
+  variant: 'light',
+  isMobile: undefined,
```

Comment:
Although it works good, is it more appropriate to set `isMobile` to `false` by default instead of `undefined`?

Reply:
Fixed. Thanks!